assets/src/scss/layout/_cookies-new.scss

```diff
@@ -1,16 +1,35 @@
+@mixin slide-from($direction, $margin) {
+  $animation-name: unique-id() !global;
+
+  @keyframes #{$animation-name} {
+    0% {
+      #{$direction}: -100%;
+    }
+
+    100% {
+      #{$direction}: $margin;
+    }
+  }
+
+  &.shown {
+    animation: 1s $animation-name;
+  }
+}
+
 .cookie-notice.new-design {
   z-index: 100;
   position: fixed;
-  bottom: 0;
   width: 100vw;
   background: white;
-  padding: 26px 24px 32px 24px;
+  padding: 24px 24px 32px 24px;
```

Comment:
Just a suggestion, you could replace to a padding shorthand `padding: 24px 24px 32px` (top|**right & left|**bottom|)

Reply:
I agree that is less clear, however and thinking about the performance it'll be improve the bundle size, why? because are at least 5 characters less and it's equal to at least 5 bytes less 🤓

Reply:
Of course that it won't affect the performance in this case but if you change just this in the rest of files I assume that it will.

Reply:
Okay, got it! That's a fair point, I've updated the code 🙂